wasm/lib/src/js_module.rs

```diff
@@ -386,20 +375,14 @@ pub struct PyPromise {
 }
 pub type PyPromiseRef = PyRef<PyPromise>;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PyValue)]
```

Comment:
PyValue is expected to be moved to PyPromise but to PromiseKind

Reply:
Yes. I will fix in new commit.
Thanks!